Gemfile

```diff
@@ -2,5 +2,6 @@ source 'https://rubygems.org'
 
 gem 'rails-controller-testing'
 gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'ckeditor', github: 'galetahub/ckeditor', branch: 'master', ref: '4385f23b90f84c25f7b912bd75c233c8f27b5f58'
```

Comment:
@PSzyszkaSpark we don't need to use the master branch,. 5.0.0 is already released